### ISSUE TITLE
pdflatex : Package calc Error: `.' invalid at this point.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halamalala/inkscape2tikz#21]
```
When i do :

svg2tikz bulle_1.svg 

svg file output is :

\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage{tikz}

\begin{document}

\begin{tikzpicture}[y=0.80pt,x=0.80pt,yscale=-1, inner sep=0pt, outer sep=0pt]

\path[cm={{-1.2838419,0.26777553,-1.1244913,-0.58595252,(1175.3965,253.66694)}},
draw=black,miter
    limit=4.00,line width=7.795pt]
    (547.4733,328.4836)arc(-367.887:-26.815:216.677720 and 147.987);
  \path[draw=black,line join=miter,line cap=round,miter limit=4.00,line
    width=8.000pt] (182.6222,229.3359) -- (50.2923,316.2091);
  \path[draw=black,line join=miter,line cap=round,miter limit=4.00,line
    width=8.000pt] (101.6315,206.6994) -- (49.2821,313.1786);

\end{tikzpicture}
\end{document}


But when i made pdflatex of this, I get :

! Package calc Error: `.' invalid at this point.

On fedora 11 with last version of 
inkscape2tikz 


Other info :

[root@localhost inkscape2tikz]# pdflatex --version
pdfTeX using libpoppler 3.141592-1.40.3-2.2 (Web2C 7.5.6)
kpathsea version 3.5.6
Copyright 2007 Peter Breitenlohner (eTeX)/Han The Thanh (pdfTeX).
Kpathsea is copyright 2007 Karl Berry and Olaf Weber.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX using libpoppler copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX using libpoppler source.
Primary author of pdfTeX using libpoppler: Peter Breitenlohner (eTeX)/Han
The Thanh (pdfTeX).
Kpathsea written by Karl Berry, Olaf Weber, and others.

Compiled with libpng 1.2.37; using libpng 1.2.37
Compiled with zlib 1.2.3; using zlib 1.2.3
Compiled with libpoppler

```

Original issue reported on code.google.com by `magepe...@gmail.com` on 19 Jan 2010 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
What version of PGF are you using? The generated code compiles fine using PGF 
2.0 and
a recent development version of PGF from http://www.texample.net/tikz/builds/

The problem could be that you are running PGF 1.18. svg2tikz only supports PGF 
2.0 or
newer. You will find this information in the log file generated during 
compilation. 

Unfortunately the graphic is not rendered correctly in PGF 2.0, so I need to
investigate this further. 

```

Original comment by `kjel...@gmail.com` on 19 Jan 2010 at 1:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
In fedora 11, texlive 2007 is used.
I gonna try with texlive 2009.

About pgf version, I'm not sure about this, but when i do this :

 cat test.log | grep revision

I get :
Package: pgfrcs 2006/10/26 v1.10 (rcs-revision 1.14)
Package: pgf 2006/10/11 v1.10 (rcs-revision 1.7)
Package: pgfsys 2006/10/16 v1.10 (rcs-revision 1.19)
File: pgf.cfg 2006/10/11  (rcs-revision 1.2)
File: pgfsys-pdftex.def 2006/10/23  (rcs-revision 1.14)
File: pgfsys-common-pdf.def 2006/10/16  (rcs-revision 1.7)
File: pgfsyssoftpath.code.tex 2006/10/16  (rcs-revision 1.4)
File: pgfsysprotocol.code.tex 2006/10/16  (rcs-revision 1.4)
Package: pgfcore 2006/10/11 v1.10 (rcs-revision 1.4)
File: pgfcoremath.code.tex 2006/10/15  (rcs-revision 1.3)
File: pgfcorepoints.code.tex 2006/10/16  (rcs-revision 1.7)
File: pgfcorepathconstruct.code.tex 2006/10/16  (rcs-revision 1.6)
File: pgfcorepathusage.code.tex 2006/10/16  (rcs-revision 1.7)
File: pgfcorescopes.code.tex 2006/10/23  (rcs-revision 1.20)
File: pgfcoregraphicstate.code.tex 2006/10/16  (rcs-revision 1.6)
File: pgfcoretransformations.code.tex 2006/10/11  (rcs-revision 1.4)
File: pgfcorequick.code.tex 2006/10/11  (rcs-revision 1.2)
File: pgfcoreobjects.code.tex 2006/10/11  (rcs-revision 1.2)
File: pgfcorepathprocessing.code.tex 2006/10/16  (rcs-revision 1.5)
File: pgfcorearrows.code.tex 2006/10/16  (rcs-revision 1.6)
File: pgfcoreshade.code.tex 2006/10/16  (rcs-revision 1.7)
Package: pgfbaseshapes 2006/10/16 v1.10 (rcs-revision 1.16)
Package: pgfbaseplot 2006/10/16 v1.10 (rcs-revision 1.5)
Package: pgfbaseimage 2006/10/16 v1.10 (rcs-revision 1.5)
Package: pgfbaselayers 2006/10/16 v1.10 (rcs-revision 1.5)
Package: pgfbasesnakes 2006/10/16 v1.10 (rcs-revision 1.10)
Package: pgfbasepatterns 2006/10/16 v1.10 (rcs-revision 1.9)
Package: pgfcomp-version-0-65 2006/10/11 v1.10 (rcs-revision 1.4)
Package: pgffor 2006/10/16 v1.10 (rcs-revision 1.5)
Package: tikz 2006/10/17 v1.10 (rcs-revision 1.68)
File: pgflibraryplothandlers.code.tex 2006/10/16 v1.10 (rcs-revision 1.8)
File: pgflibrarytikztopaths.code.tex 2006/10/16 v1.10 (rcs-revision 1.5)

```

Original comment by `magepe...@gmail.com` on 20 Jan 2010 at 8:45

**Comment by GoogleCodeExporter** [comment]
```
It looks like Texlive 2007 ships with PGF 1.10. For the latest PGF features I 
highly
recommend installing a recent development version from
http://www.texample.net/tikz/builds/
```

Original comment by `kjel...@gmail.com` on 20 Jan 2010 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
With texLive 2009 (futur rpm for F13 .. ) pdflatex work.

But, like you say above, image render isn't good, I add svg file.
```

Original comment by `magepe...@gmail.com` on 20 Jan 2010 at 9:38

Attachments:
- [bulle_1.svg](https://storage.googleapis.com/google-code-attachments/inkscape2tikz/issue-21/comment-4/bulle_1.svg)

**Comment by GoogleCodeExporter** [comment]
```
The problem is that PGF 2.0 does not like angles larger that 360deg. This means 
that
we have to normalize the angle to the [-360, 360] range. In principle it should 
be
enough with angle modulo 360, but I need to figure out how to handle negative 
angles
correctly.  
```

Original comment by `kjel...@gmail.com` on 27 Jan 2010 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5520616360.
```

Original comment by `kjel...@gmail.com` on 31 Jan 2010 at 6:10
- Changed state: **Fixed**
